phx/events/admin.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib import admin
+
+from phx.admin import phx_admin
+
+from .models import Event
+
+
+class EventsAdmin(admin.ModelAdmin):
+    list_display = ['name', 'location']
+    exclude = ['uploaded_by']
+
+    def save_model(self, request, obj, form, change):
+        if getattr(obj, 'uploaded_by', None) is None:
+            obj.uploaded_by = request.user
+        obj.save()
```

Comment:
I'm not sure if we need this, although it's fine to leave it in. It was added initially so we could keep track of who (manually) added a news story or page, but if events (etc) are going to be added via a cron it'll usually be empty.